Src/ILGPU/IR/Values/Terminators.cs

```diff
@@ -275,6 +279,17 @@ internal Branch(in ValueInitializer initializer)
         protected override TypeNode ComputeType(in ValueInitializer initializer) =>
             initializer.Context.VoidType;
 
+        /// <summary cref="Value.Write{T}(T)"/>
+        protected internal override void Write<T>(T writer)
+        {
+            int index = 0;
+            writer.Write(nameof(NumTargets), NumTargets);
+            foreach (var target in Targets)
+            {
```

Comment:
NIT: Please remove braces that are not required to match the style of the code base